include/sway/desktop/fx_renderer/fx_renderer.h

```diff
@@ -165,6 +167,19 @@ void fx_renderer_clear(const float color[static 4]);
 
 void fx_renderer_scissor(struct wlr_box *box);
 
+// Initialize the stenciling work
+void fx_renderer_stencil_mask_init();
+
+// Close the mask
+void fx_renderer_stencil_mask_close(bool draw_inside_mask);
+
+// Finish stenciling and clear the buffer
+void fx_renderer_stencil_mask_fini();
+
+void fx_renderer_stencil_enable();
+
+void fx_renderer_stencil_disable();
+
```

Comment:
these dont need to be in the header since they're only used inside fx_renderer.c